search.go

```diff
@@ -49,6 +49,7 @@ func search(token string) error {
 		for _, s := range statuses {
 			if strings.Contains(strings.ToLower(cleanupContent(s.Content)), token) {
 				fmt.Println("\nFound toot:", cleanupContent(s.Content))
+				fmt.Println("Date: ", s.CreatedAt, ", URL: ", s.URL)
```

Comment:
May be a good idea to format the time string a bit, here. Also, the spaces are redundant, fmt.Println will already add one between each parameter.

Reply:
I changed it to RFC822 now and URL and time are in separate lines now. I think that's a bit more visually appealing.